glusterd2/main.go

```diff
@@ -119,6 +121,29 @@ func main() {
 		log.WithError(err).Fatal("Failed to generate local auth token")
 	}
 
+	// Get the Jaeger endpoints from config file
```

Comment:
Please move all the jaeger initialization into a seperate package or a seperate file in the main package if you feel creating a new package is a bit much.
I'd prefer a seperate package, so that you could reuse the configuration and intialization for glustercli as well.

Reply:
Agreed. I have moved the jaeger initialization to a separate package called 'tracing' within glusterd2/pkg. Please check latest evidence.